test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -113,7 +114,7 @@ public class RazorPageTest
             await page.ExecuteAsync();
 
             // Assert
-            var pageOutput = page.Output.ToString();
```

Comment:
This is just cleanup. This code assumed that `.Output` was always a `StringWriter`